src/main/java/oop/Customer.java

```diff
@@ -0,0 +1,45 @@
+package oop;
+
+import java.util.Scanner;
+
+public class Customer {
+    private String name;
+    private int age;
+
+    public Customer(String name, int age) {
+        this.name = name;
+        this.age = age;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    public void setAge(int age) {
+        this.age = age;
+    }
+
+    Scanner scan = new Scanner(System.in);
+
+
+    public Object selectHall(){
```

Comment:
do not use Object as a return type, it is too general and may cause issues with casting
same for choosePreferFilm() method